### ISSUE TITLE
too few test samples 

### ISSUE THREAD

**Issue by makovez** [chathumal93/Pytorch-UNet-Flood-Segmentation#1]
Hi, I think your result of 0.87 IoU might be influenced by the few samples number in test set:
Your overall split train/test is 97.6% train and 2.4% test.
Have you tried decreasing train and increasing test samples ?

**Comment by chathumal93** [comment]
Train, validation and  test percentages are respectively 88%, 10% and 2%. Test dataset is small as you have observed and the test dataset is selected in a way that the water percentage ranges from 0-100 percent.  I haven't tried reducing the training dataset, since the whole dataset is small.
